libursa/src/wasm/bls.rs

```diff
@@ -8,14 +8,17 @@ pub struct Generator(bls::Generator);
 
 #[wasm_bindgen]
 impl Generator {
+    #[wasm_bindgen(constructor)]
     pub fn new() -> Result<Generator, JsValue> {
         Ok(Generator(maperr!(bls::Generator::new())))
     }
 
-    pub fn as_bytes(&self) -> Vec<u8> {
+    #[wasm_bindgen(js_name = toBytes)]
+    pub fn to_bytes(&self) -> Vec<u8> {
```

Comment:
I would expect changing the name from `as_bytes` to `to_bytes` would cause problems elsewhere, is that not the case?

Reply:
It would change it to a breaking change if the WASM module is used anywhere. Internally it still uses `as_bytes().to_vec()`.

I mainly changed it to be more inline with the JS way of doing a transformation. We do not really have an `as` way of doing it like this. 

TBH, I would be happy to change it back to be more inline with the rust interfaces.

Reply:
If it doesn't break anything, I'm not bothered. Thanks for clarifying.